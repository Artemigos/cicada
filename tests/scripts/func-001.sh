function foo-bar() {
    echo foo
    echo bar
}

function ifif7 {
    echo 27 if printed
}

function what-is-my-args {
    echo $@
    echo $0 $1 $2 $3
}

foo-bar
ifif7

what-is-my-args a b c d
echo current script is $0 | sed 's|\(.*\) [^ ]*\(tests/scripts/func-001.sh\)|\1 \2|'
