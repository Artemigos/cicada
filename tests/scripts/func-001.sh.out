foo
bar
27 if printed
a b c d
what-is-my-args a b c
current script is tests/scripts/func-001.sh
